enterprise/models.py

```diff
@@ -453,6 +453,8 @@ def get_tpa_hint(self, tpa_hint_param=None):
         # if there is only one identity provider linked with the customer and tpa_hint_param was not provider.
         if self.has_single_idp and not tpa_hint_param:
             return self.identity_providers.first().provider_id
+        if self.has_multiple_idps and not tpa_hint_param:
+            return self.default_provider_idp.provider_id
```

Comment:
For the safe side we should check if the `self.default_provider_idp` exists before excessing its `provider_id`.

Reply:
If an enterprise wants to link more than one IDP then it has to made default one of them. Otherwise, it can not link it.

Reply:
<img width="1669" alt="Screenshot 2021-06-22 at 2 31 45 PM" src="https://user-images.githubusercontent.com/11922730/122901110-abf4bc80-d366-11eb-83a3-10c86b57d509.png">

Reply:
@zamanafzal You are 100% right, we can't link multiple IDPs without defining default from the admin, But this restriction is only in admin view. But this constraint is not on the Model level. In future, there might be scenarios when we link multiple IDPs with default so I would recommend better checking if default IDP is there before using it.

Reply:
I think this behaviour is not expected if we look at the default IDP ticket. But still for the safe side, I have added this check and test case accordingly.